spec/bowling_spec.rb

```diff
@@ -8,9 +8,22 @@
     expect(bowling.score).to eq 10
```

Comment:
Nice testing here! You could consider wrapping your tests for #roll within its own describe block, though this is a matter of preference really. All in all though great so far!